corgidrp/mocks.py

```diff
@@ -102,382 +103,315 @@
 
 def create_default_L1_headers(arrtype="SCI", vistype="TDEMO"):
     """
-    Creates an empty primary header and an Image extension header with currently
-        defined keywords.
-
+    Creates default L1 headers by reading values from the l1.rst documentation file.
+    
     Args:
-        arrtype (str): Array type (SCI or ENG). Defaults to "SCI". 
-        vistype (str): Visit type. Defaults to "TDEMO"
-
+        arrtype (str): Array type ("SCI" or "ENG"). Defaults to "SCI".
+        vistype (str): Visit type. Defaults to "TDEMO".
+    
     Returns:
-        tuple:
-            prihdr (fits.Header): Primary FITS Header
-            exthdr (fits.Header): Extension FITS Header
-
-    """
-    dt = datetime.datetime.now(datetime.timezone.utc)
-    dt_str = dt.isoformat() 
+        tuple: 
+            prihdr (fits.Header): Primary FITS header with L1 keywords
+            exthdr (fits.Header): Extension FITS header with L1 keywords
     
+    """
+    # Create empty headers
     prihdr = fits.Header()
     exthdr = fits.Header()
+    
+    # Set up dynamic values
+    dt_str = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
+    # Override NAXIS values to match test expectations (1024x1024)
+    # L1.rst documents actual detector dimensions (2200x1200 for SCI, 2200x2200 for ENG)
+    NAXIS1 = 1024
+    NAXIS2 = 1024
+    
+    def parse_rst_table(rst_file_path, section_name, stop_at=None):
```

Comment:
due to how python packaging works, we can't have our code depend on the docs (docs can only depend on code). For any use that installs the code not in developer mode (i.e., the CTC I believe does this), the docs folder is not "installed". Thus, we cannot use the .rst table. Unfortunately, we'll need to use the previous code we had.